src/test/java/com/auth0/jwk/JwkTest.java

```diff
@@ -155,4 +235,17 @@ private static String randomKeyId() {
         values.put("e", EXPONENT);
         return values;
     }
+
+    private static Map<String, Object> publicKeyEllipticCurveValues(String kid, Object keyOps, String crv, String x, String y) {
+        Map<String, Object> values = Maps.newHashMap();
+        values.put("alg", ES_256);
```

Comment:
If the curve changes to P-256, P-384, P-521. Shouldn't the algorithm name also change and have the same size?

Reply:
Done.